### PR TITLE
Speed up shield sync

### PR DESCRIPTION
**Pull request by panleone**
## Abstract

Do not fetch coinbase and coinstake txs when syncing shield. It's useless since they will never contain shield data.
This simple optimization made my wallet sync 40% faster.

### CODE REVIEW

**Review by JSKitty — APPROVED** [review]
**tACK** 5c031d15f1e57de2741098582838dd12813c92cd

I tested this back-to-back against another `Master` instance using the same seed - unfortunately my internet is too slow to see a huge difference, but I can still notice it. Either way, code looks good and makes sense.

File: scripts/network.js [hunk 1]
```diff
@@ -127,22 +127,26 @@ export class ExplorerNetwork extends Network {
...
      * @returns {Promise<Object>} the block fetched from explorer
      */
-    async getBlock(blockHeight) {
+    async getBlock(blockHeight, skipCoinstake = false) {
         try {
             const block = await this.safeFetchFromExplorer(
                 `/api/v2/block/${blockHeight}`
             );
             const newTxs = [];
+            const txsToAdd = block.txs;
```
Comment (Duddino) [inline scripts/network.js:140]: Variable is unused

File: scripts/network.js [hunk 2]
```diff
@@ -127,10 +127,11 @@ export class ExplorerNetwork extends Network {
 
     /**
      * Fetch a block from the explorer given the height
-     * @param {Number} blockHeight
+     * @param {Number}
+     * @param {skipCoinstake} - if true coinstake tx will be skipped
```
Comment (Duddino) [inline scripts/network.js:131]:
```suggestion
     * @param {number} blockHeight
     * @param {boolean} skipCoinstake - if true coinstake tx will be skipped
```

**Review by Duddino — DISMISSED** [review]
tACK other than minor JSdoc nits

**Review by JSKitty — APPROVED** [review]
**tACK** 6644ac1b972f5b6d302df842e62ca13dffc4743d

**Pull request merged**